addon-test-support/-private/mock-server.js

```diff
@@ -9,7 +9,7 @@ let createMock = async function(path, method, statusCode, response) {
     path = `${url.pathname}${url.search}`;
   }
 
-  return await fetch('/__mock-request', {
+  return fetch('/__mock-request', {
```

Comment:
`fetch` returns a promise, so does not seem make any sense to have `await` as no result or error handling takes place